src/Raven.Server/ServerWide/Commands/Subscriptions/AcknowledgeSubscriptionBatchCommand.cs

```diff
@@ -72,33 +71,15 @@ protected override BlittableJsonReaderObject GetUpdatedValue(long index, RawData
                 return context.ReadObject(existingValue, SubscriptionName);
             }
 
-            if (IsLegacyCommand())
```

Comment:
why?

Reply:
I think we won't have mixed cluster 6.x with version  5.2 or older

Reply:
yes, but we might have old not executed raft command in the state machine.

Reply:
reverted